src/buttonDelivery.styles.js

```diff
@@ -0,0 +1,8 @@
+export const buttonDetailStyle = {
+  color: "#000",
+  textTransform: "none",
+  boxShadow: `0px 1px 4px  rgba(0, 0, 0, 0.15)`,
+  minWidth: 122,
+  minHeight: 40,
```

Comment:
this hasn't been resolved, the button doesn't need a min-height nor a height, the height is calculated automatically
```suggestion

```

Reply:
ok